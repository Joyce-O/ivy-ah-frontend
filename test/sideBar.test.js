import React from 'react';
import renderer from 'react-test-renderer';

const onCloseMenu = jest.fn();
const setMenuRef = jest.fn();
const closeMenu = jest.fn();

it('renders the side bar', () => {
  const sideBar = renderer
    .create(<sideMenu
      onCloseMenu={onCloseMenu}
      setMenuRef={setMenuRef}
      closeMenu={closeMenu}
    />)
    .toJSON();
  expect(sideBar).toMatchSnapshot();
});
