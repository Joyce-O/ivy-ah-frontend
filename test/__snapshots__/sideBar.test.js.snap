// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the side bar 1`] = `
<sideMenu
  closeMenu={[MockFunction]}
  onCloseMenu={[MockFunction]}
  setMenuRef={[MockFunction]}
/>
`;
